test/parallel/test-icu-locale.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+const common = require('../common');
+const { spawnSyncAndExitWithoutError } = require('../common/child_process');
+
+if (!common.hasIntl)
+  common.skip('Intl not present.');
+
+if (!common.isMainThread)
+  common.skip('process.env.LOCALE is not intercepted in Workers');
```

Comment:
```suggestion
  common.skip('process.env.NODE_OPTIONS is not intercepted in Workers');
```
...although I don't know if that's actually true.

Reply:
You are right: #31711